.github/CODEOWNERS

```
@@ -1,4 +1,4 @@
 # CODEOWNERS: https://help.github.com/articles/about-codeowners/
 
 # Primary repo maintainers
-*       @alessio @alexanderbez @jgimeno @shahankhatch @zmanian @yaruwangway
+*       @okwme @alexanderbez @zmanian @yaruwangway @marbar @crodriguezvega @cmwaters @jackzampolin
```

```suggestion
*       @okwme @alexanderbez @zmanian @yaruwangway @crodriguezvega @cmwaters @jackzampolin
```

I dont review things often here, I only show up when pinged. Don't thinks it's fair to others that show up regularly to include me

---

it will be helpful for the community to assign reviewers for docs, whoever those reviewers are

suggest adding *.md to the file